bin/delete-everything-split-repo.sh

```diff
@@ -36,9 +36,18 @@ git checkout -b 'deleteMostOfLibrary'
 shopt -s extglob
 
 # Delete everything first that's not the samples, README.md, .kokoro folder, and git folder
-rm -rf !("samples"|"README.md"|".git"|".kokoro") 
-rm -rf "/tmp/${SPLIT_REPO}/.[!.git|!.kokoro]*"
+rm -rf !("samples"|"README.md"|".git") 
```

Comment:
Does this syntax actually work? The way I would do it is

```sh
ls | grep -v 'samples\|README.md\|.git' | xargs rm -f
```

Reply:
Wouldn't this say delete samples, README, and .git? The syntax I opted for should delete everything but those. I think it does work! See: https://github.com/googleapis/nodejs-language/pull/701

Reply:
I'm still interested in bash magic though, if you have another way please share :)

Reply:
I mean, I have never seen this syntax, and it did not work for me when I tried, that's why I asked :)